tools/transformer/src/transformations/oneOf.js

```diff
@@ -90,6 +90,9 @@ function transformOneOfProperties(parentObject, api) {
   );
 
   for (let childObject of childObjects) {
+    if (!childObject.properties) {
```

Comment:
This is extra check to ensure corner cases when children do not have any properties.